.github/workflows/ci.yml

```
@@ -327,3 +327,16 @@ jobs:
           prerelease: false
           files: |
             package-*/*
+            
+  winget-publish:
+    needs: release
+    if: (runner.os == 'Windows') && contains(github.ref, 'tags/v')
+    runs-on: windows-latest
+    steps:
+      - name: Submit package to Windows Package Manager Community Repository
```

```suggestion
      - name: Submit
```

I know this seems picky, but long names can be a bit of a distraction in the workflow summary.